packages/language-server/test/plugins/typescript/features/UpdateImportsProvider.test.ts

```diff
@@ -42,21 +44,52 @@ describe('UpdateImportsProviderImpl', () => {
     afterEach(() => sinon.restore());
 
     it('updates imports', async () => {
-        const { updateImportsProvider, fileUri } = await setup('updateimports.svelte');
+        const { updateImportsProvider, fileUri } = await setup(
+            'updateimports.svelte',
+            ts.sys.useCaseSensitiveFileNames
+        );
 
         const workspaceEdit = await updateImportsProvider.updateImports({
-            // imported files both old and new have to actually exist, so we just use some other test files
-            oldUri: pathToUrl(join(testFilesDir, 'diagnostics', 'diagnostics.svelte')),
+            oldUri: pathToUrl(join(testFilesDir, 'imported.svelte')),
```

Comment:
This comment seems to be outdated. It now seems to be able to check even if the new file does not exist.